doc/release-notes.md

```diff
@@ -10,6 +10,22 @@ Option handling
 - The `-reindex` and `-reindex-chainstate` options now imply `-rescan`
   (provided that the wallet is enabled and pruning is disabled, and unless
   `-rescan=0` is specified explicitly).
+- A new `-anchorconfirmations` argument has been added to allow the user
+  to specify the number of blocks back from the chain tip that anchors 
+  will be selected from when spending notes. By default, anchors will
+  now be selected at a depth of 10 blocks from the chain tip. Values
```

Comment:
Is there an off-by-one error in this description? If 1 indicates the block at the tip (i.e. 1 confirmation), then 10 confirmations indicates 9 blocks back from the tip, not 10.